versions/3.0.md

```diff
@@ -324,7 +324,7 @@ An object representing a Server.
 Field Name | Type | Description
 ---|:---:|---
 <a name="serverUrl"></a>url | `string` | A URL to the target host.  This URL supports Server Variables and may be relative, to indicate that the host location is relative to the location where the OpenAPI definition is being served. Variable substitutions will be made when a variable is named in `{`brackets`}`.
-<a name="serverDescription"></a>description | `string` | An optional string describing the host designated by the URL.
+<a name="serverDescription"></a>description | `string` | An optional string describing the host designated by the URL. [CommonMark syntax](http://spec.commonmark.org/) can be used for rich text representation.
```

Comment:
I think it is ambiguous and can/should be simplified (more concise, not ambiguous). I'm not referring to the use of the word "optional", but the phrase "can be used", which implies something other than CommonMark can be used if the author does not want rich text. This *is* CommonMark.